.gitignore

```diff
@@ -79,6 +79,7 @@ fabfile/
 .sass-cache/
 node_modules/
 bower_components/
+frontend/index.html
 
```

Comment:
Yep, do this please! It will be helpful and more meaningful for other contributors.